hamilton/data_quality/default_validators.py

```diff
@@ -508,6 +508,23 @@ def _append_pandera_to_default_validators():
 _append_pandera_to_default_validators()
 
 
+def _append_pydantic_to_default_validators():
+    """Utility method to append pydantic validators as needed"""
+    try:
+        from pydantic import BaseModel  # noqa: F401
+    except ModuleNotFoundError:
+        logger.info(
```

Comment:
Let's make this a debug? Also, maybe import pydantic rather than BaseModel, as I don't think we use it?